frontend/src/redux/museumArtworkActions.js

```diff
@@ -15,6 +15,62 @@ const convertToArtworkInfo = (artwork) => {
   return artworkInfo;
 };
 
+function filterArtworks(artworks) {
+  const artworksArr = [];
+  for (const artwork of artworks) {
+    if (artwork.thumbnail != null) {
+      artworksArr.push(convertToArtworkInfo(artwork));
+    }
+  }
+  return artworksArr;
+}
+
+// based off of https://github.com/kjschmidt913/AIC/blob/master/script.js
+function getQuery(limit) {
+  return {
+    'resources': 'artworks',
+    'fields': [
+      'pagination',
+      'id',
+      'title',
+      'image_id',
+      'thumbnail',
+    ],
+    'limit': limit,
+    'query': {
+      'bool': {
+        'must': [
+          {
+            'term': {
+              'is_public_domain': true,
+            },
+          },
+          {
+            'exists': {
+              'field': 'image_id',
+            },
+          },
+          {
+            'exists': {
+              'field': 'thumbnail.width',
```

Comment:
Would this cause a problem if thumbnail is null?

Reply:
The [exists query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-exists-query.html) checks to see if the specified field is not null (which is helpful for making sure that an artwork has a description) and it checks that it exists at all (which is helpful in this case so if thumbnail is null, then this entire field would not exist).  The query only returns items where the specified fields exist.

Reply:
thanks for the link!